src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -870,19 +909,14 @@ namespace
     }
 }
 
-bool MainWindow::can_close_project()
+bool MainWindow::attempt_close_project()
```

Comment:
The name of this method is misleading: it isn't closing the project, it's just checking if the project can be closed. Maybe `ask_can_close_project()`? Not ideal but better. Maybe you have a better suggestion? :)